doc/rippled-example.cfg

```diff
@@ -289,16 +289,25 @@
 #   server_max_window_bits = [9..15]
 #   client_no_context_takeover = <flag>
 #   server_no_context_takeover = <flag>
-#   compress_level = [1..9]
-#   memory_level = [1..9]
 #
-#       These optional settings control options related to the
-#       permessage-deflate extension negotiation. For precise
-#       definitions of these fields please see the RFC 7692,
-#       "Compression Extensions for WebSocket":
+#       These optional settings control options related to the permessage-deflate
+#       extension negotiation. For precise definitions of these fields please see
+#       the RFC 7692, "Compression Extensions for WebSocket":
 #           https://tools.ietf.org/html/rfc7692
 #   
+#   compress_level = [0..9]
+#
+#       When set, determines the amount of compression attempted, where 0 is
+#       the least amount and 9 is the most amount. Higher levels require more
+#       CPU resources. Levels 1 through 3 use a fast compression algorithm,
+#       while levels 4 through 9 use a more compact algorithm which uses more
+#       CPU resources. If unspecified, a default of 3 is used.
```

Comment:
I don't know. I rather not repeat all the information in the RFC, because its complicated and error prone. If people are hell bent on customizing it they can decipher the RFC for themselves (which specifies a default).